### PR TITLE
Add JavaScript

### PR DESCRIPTION
**Pull request by Thinus01**
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,12 +6,86 @@
...
+    color: #000;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    opacity: 1;
+    line-height: 270%;
+    text-align: center;
+    font-weight: 300;
+    font-size: large;
+    z-index: 1;
+  }
```
Comment (Qoosim) [inline style.css:37]: - [ ] Kindly stick to the guiding template as much as possible. Take a look at the both designs and work your design alongside the guiding template. 

### Your Design
your-design

### Figma Design
figma

**Review by Qoosim — CHANGES_REQUESTED** [review]
#   *Status: Required Changes* :recycle: 

Hi @Thinus01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly update your README file with this [new template](https://github.com/microverseinc/readme-template)

File: style.css [hunk 2]
```diff
@@ -6,10 +6,114 @@
...
+    top: 0.1px;
+    bottom: 0.1px;
+    width: 100%;
+    height: 614.6%;
+    background: rgb(90, 151, 243);
+    backdrop-filter: blur(5px);
+    color: #6070ff;
+    mix-blend-mode: multiply;
+    transition: 50ms;
+  }
```
Comment (Gopxfs) [inline style.css:28]: Hello @Thinus01, nice job updating your menu design 😄. Although, as requested by the previous reviewer, you still have room for improvement. Check below:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Important points:
- The font-weight of the links (`Portfolio` `About` `Contact`) is higher in the original design (600).
- Notice how the original background has a strong blur on the whole page, highlighting the links.

You are almost there! Cheers and keep coding 😄.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @Thinus01 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @Thinus01 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**